exercises/Scientific-0-R.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: exercise
+title: Scientific 0
+language: R
+---
+
+*This problem preceeds [Combining Basics]({{ site.baseurl }}/exercises/Combining—the-basics-R)*
+
+Dr. Granger is interested in studying the factors controlling the size and
+carbon storage of shrubs. This research is part of a larger area of research
+trying to understand carbon storage by plants. She has conducted a small
+preliminary experiment looking at the effect of three different treatments on
+shrub volume at four different locations. She wants to conduct a preliminary
+analysis of these data to include in a grant proposal and she would like you to
+conduct the analysis for her (she might be a world renowned expert in carbon
+storage in plants, but she sure doesn't know much about computers). She has
+placed a data file on the web for you to
+[download]({{ site.baseurl }}/data/shrub_volume_experiment.csv).
+
+You want to get familiar with the data so…
+
+1. Import the shrub dimensions data using `read.csv()`. It has a header row so 
+you'll need to tell R there are column names by providing the optional argument 
+`head = TRUE`.
```

Comment:
>> Seems like read.csv() assumes there is a header, so we shouldn't need
the second sentence.
Sentence removed.

On Thu, Jun 11, 2015 at 3:54 PM, Ethan White <notifications@github.com>
wrote:

> In exercises/Scientific-0-R.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/60#discussion_r32267013>
> :
>
> > +Dr. Granger is interested in studying the factors controlling the size and
> > +carbon storage of shrubs. This research is part of a larger area of research
> > +trying to understand carbon storage by plants. She has conducted a small
> > +preliminary experiment looking at the effect of three different treatments on
> > +shrub volume at four different locations. She wants to conduct a preliminary
> > +analysis of these data to include in a grant proposal and she would like you to
> > +conduct the analysis for her (she might be a world renowned expert in carbon
> > +storage in plants, but she sure doesn't know much about computers). She has
> > +placed a data file on the web for you to
> > +[download]({{ site.baseurl }}/data/shrub_volume_experiment.csv).
> > +
> > +You want to get familiar with the data so…
> > +
> > +1. Import the shrub dimensions data using `read.csv()`. It has a header row so
> > +you'll need to tell R there are column names by providing the optional argument
> > +`head = TRUE`.
>
> Seems like read.csv() assumes there is a header, so we shouldn't need the
> second sentence.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/60/files#r32267013>
> .
>